clap_mangen/src/render.rs

```diff
@@ -212,6 +214,7 @@ fn long_option(opt: &str) -> Inline {
     bold(&format!("--{}", opt))
 }
 
+#[cfg(feature = "env")]
 fn option_environment(opt: &clap::Arg) -> Option<Vec<Inline>> {
```

Comment:
I had missed this when fixing some of the other feature flags.  In my opinion, stable but optional features should have no-op reflection functions when disabled, so instead we'd offer a version of `get_env` that returns `None` and `is_hide_env_set` returns `true` when `env` is not set.

I'm also tempted to just enable it.  It doesn't really buy us much in terms of compile times or binary size and as we modularize, it'll shifting to being optional but without feature flags which aren't ideal to work with.